mongo/database.go

```diff
@@ -161,8 +165,13 @@ func (db *Database) processRunCommand(ctx context.Context, cmd interface{},
 		Database(db.name).Deployment(db.client.deployment).ReadConcern(db.readConcern).Crypt(db.client.crypt), sess, nil
 }
 
-// RunCommand runs a command on the database. A user can supply a custom
-// context to this method, or nil to default to context.Background().
+// RunCommand executes the given command against the database.
+//
+// The runCommand parameter should be a document for the command to be executed. It cannot be nil.
+// This must be an order-preserving type such as bson.D. Map types such as bson.M are not valid.
+// If the command document contains a session ID or any transaction-specific fields, the behavior is undefined.
+//
+// The opts parameter can be used to specify options for this operation (see the options.RunCmdOptions documentation).
```

Comment:
I'm not sure whether to say this once, or whether it should just be repeated for every method that takes options, but the docs should specify the behavior of passing more than one options argument (since it's an `opts ...`).

Reply:
I think this should be explained, but I don't see much value in adding it to every method that takes options. Maybe this could be in the top level `doc.go` file for the `options` package? I can change the `ClientOptions` ticket to be "top-level options docs + ClientOptions docs" if you agree.

Reply:
sgtm